iterations-Katharina601/gql-queries/vol_qry.r

```diff
@@ -0,0 +1,30 @@
+library(glue)
+
+#Defining the vol_gry function
+vol_qry <- function(id, from, to) {
+  query <- glue::glue('
+    {{
+      trafficData(trafficRegistrationPointId: "{id}") {{
+        volume {{
+          byHour(from: "{from}", to: "{to}") {{
+            edges {{
+              node {{
+                from
+                to
+                total {{
+                  volumeNumbers {{
+                    volume
+                  }}
+                }}
+              }}
+            }}
+          }}
+        }}
+      }}
+    }}',
+                      .open = "{{",
+                      .close = "}}"
+  )
+  
+  return(query)
+}
```

Comment:
Almost correct. 

The .open and .close arguments specifies when you will put values in the string. In your case it seems like you want .open and .close to be "{" and "}".  

Also, there should only be single curly brackets in the rest of the string, meaning there should be "}" instead of "}}"